tests/OTAnalytics/plugin_ui/test_cli.py

```diff
@@ -529,8 +803,15 @@ def test_start_with_no_video_in_folder(
             save_name=str(save_name),
             save_suffix=save_suffix,
             count_intervals=[count_interval],
+            cli_mode=mode,
         )
-        cli = OTAnalyticsCli(run_config, **cli_dependencies)
+
+        cli: OTAnalyticsCli
+        if mode == CliMode.STREAM:
```

Comment:
This if statement is duplicated in multiple tests. Please consider extracting it into a method or function